source/_components/media_player.enigma2.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: "Volumio Media Player"
+description: "How to set up the Enigma2 media player platform"
+date: 2017-04-07
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
```

Comment:
If there is no logo, please leave it out. The Home Assistant logo only be uses for Home Assistant related components/platforms.